packages/core/src/config.ts

```diff
@@ -456,8 +457,16 @@ const composeFormatConfig = (format: Format): RsbuildConfig => {
       };
     case 'cjs':
       return {
+        plugins: [pluginCjsShim()],
```

Comment:
Why using a plugin for this feature?

Reply:
Easy to remove in the future, and implement is a plugin form is way more decouple from other implementations and more easy to scale.